### ISSUE TITLE
Meta-analyze MTAG outputs?

### ISSUE THREAD

**Issue by unepanthere** [JonJala/mtag#133]
Hi Everyone,

  this is a question rather than an issue;

  Can one conduct a meta-analysis of MTAG outputs? Is this viable, or it would be like double-counting (as the SNPs put forward in the meta were already boosted by MTAG)?

Thanks!

**Comment by paturley** [comment]
Hello.

Just to make sure I understand, you are asking if it is OK to combine a set
of summary statistics using MTAG and using the resulting sets of MTAG
summary statistics to combine them again using meta-analysis? Generally,
this will not be OK. When you combine sets of summary statistics with MTAG,
the error between the sets of MTAG summary statistics is correlated, so a
standard meta-analysis that doesn't take this correlation into account
would have standard errors that are too tight.

If instead what you meant was that you wanted to take a set of MTAG summary
statistics and meta-analyze them with a different set of GWAS summary
statistics that didn't have any sample overlap with any of the cohorts used
in the MTAG analysis, then I think that should be OK.

Does that answer your question?

Best,
Patrick

**Comment by unepanthere** [comment]
Hi Patrick,

  Thank you for your prompt response.

  I think I'd fall in your first setup, although let me try to be more explicit. Suppose I have three summary GWAS from three independent cohorts of related but not identical phenotypes; let's call them S1, S2 and S3. I can then use MTAG to analyse them simultaneously, and MTAG will then provide adjusted summary GWAS for each; let's call them A1, A2 and A3. Right now I'm doing a meta-analysis of S1, S2 and S3. I was wondering if instead I meta-analyzed A1, A2 and A3 I would have better genetic signal for the SNPs pleiotropic for my phenotypes. In the paper, the follow-up analyses are on a per-phenotype basis, but not from a "combined" analysis of MTAG-adjusted results.

Very Best,
  Marc.

**Comment by paturley** [comment]
Hi Marc,

If I understand correctly what you are doing, in your case, the error of
A1, A2, and A3 will be correlated. This is because there is sample overlap
between the A1, A2, and A3 samples (since they are all based on the S1, S2,
and S3 samples). So meta-analyzing A1, A2, and A3 will result in standard
errors that are too tight. If you want to produce summary statistics that
give the combined average effect of each SNP across your three
phenotypes (so that pleiotropic SNPs with concordantly-signed effects are
more likely to be significant), then you are better off combining S1, S2,
and S3 since there is no sample overlap between them.

Am I understanding your question correctly?

Best,
Patrick

**Comment by unepanthere** [comment]
Hi Patrick,

  OK, yes.
  I think my main problem is that I seem to struggle to understand the difference between a meta-analysis and MTAG. If I perform a meta-analysis, the hope is that SNPs with concordantly-signed effects will be more likely to be significant. When I use MTAG, the SNPs with concordantly-signed effects are "boosted" in each summary GWAS, and that's the output of MTAG?

  Thanks again!

**Comment by paturley** [comment]
You can think of MTAG as a generalization of meta-analysis. In fact, if the
samples don't overlap, the traits are perfected genetically correlated, and
they have the same heritability, MTAG is equivalent to IVW meta-analysis.

**Comment by anglixue** [comment]
Hi @paturley 

I am reading this old thread. I was wondering for a typical MTAG result, are the SNP effects still marginal? or they became joint effects which takes LD into account?

If like you said, under the scenario where the samples don't overlap and perfected correlated/same heritability and MTAG is equivalent to IVW meta-analysis, I believe the MATG effects are still marginal?

**Comment by paturley** [comment]
Yes, that's right. The output of MTAG are marginal effects with the same
interpretation as what you'd get from a GWAS.
